tests/unit/test_column_group.py

```diff
@@ -1,10 +1,30 @@
 import cudf
 import pytest
+from cudf.tests.utils import assert_eq
 
 from nvtabular import ColumnGroup, Dataset, Workflow
 from nvtabular.ops import Categorify, Rename
 
 
+def test_column_group_select(tmpdir):
```

Comment:
Doesn't look like this test uses `tmpdir`

Reply:
nice catch! fixed